packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -608,4 +608,11 @@ export default class QuanticNumericFacet extends LightningElement {
       focusTarget.focus();
     }
   }
+
+  /**
+   * Sets the component in the initialization error state.
+   */
+  setInitializationError() {
```

Comment:
I think the declaration for `hasInitializationError ` boolean is missing in this file?